Nodejs/Product/TestAdapterNetStandard/TestAdapterNetStandard.csproj

```diff
@@ -78,6 +78,18 @@
     <Content Include="..\TestAdapter\TestFrameworks\Jest\jestReporter.js" Link="TestFrameworks\Jest\jestReporter.js">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\TestAdapter\TestFrameworks\Angular\Angular.js" Link="TestFrameworks\Angular\Angular.js">
```

Comment:
These don't seem to be new - were they just missed?

Reply:
They where not. This are links to the existing `js` files to add support for .NET Core. Adding them in this project will include them on the nuget package generated.